### PR TITLE
Bulk download component(POC).

### PR DESCRIPTION
**Pull request by dhaneshnm**
Bulk download component(POC) and the nexus-sdk support for archive end point.

    - Adds support to archive end point in SDK.
    - Adds a file download component to demo the use case for archive end point.
    - Integrates file download to Resource Details
    - makes adjustments to Detail components to look up and display files.

### CODE REVIEW

File: apps/search-poc/src/containers/__tests__/FileDownLoad.test.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            fetch,
+            uri: 'test',
+            links: [setToken],
+          });
+        const mockCallBack = jest.fn();
+        const props = {
+            fileIds: ['path/org/project/test-id'],
+            isDownload: true,
+            downloadFileName: 'test-download',
+            OnShowFiles: mockCallBack
```
Comment (Stafeeva) [inline apps/search-poc/src/containers/__tests__/FileDownLoad.test.tsx]: Trailing comma here?

File: apps/search-poc/src/containers/__tests__/FileDownLoad.test.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import * as renderer from "react-test-renderer";
+import FileDownload from '../FileDownLoad';
+import React from "react";
+import {  mount } from "enzyme";
```
Comment (Stafeeva) [inline apps/search-poc/src/containers/__tests__/FileDownLoad.test.tsx]: We could use single quotes here to be consistent, also here is an extra space before 'mount'

File: apps/search-poc/src/components/ResultTable.tsx [hunk 3]
```diff
@@ -25,14 +27,39 @@ const ResultsTable: React.FunctionComponent<ResultTableProps> = ({
...
   const [searchValue, setSearchValue] = React.useState();
+  const [selectAll, setSelectAll] = React.useState<boolean>(false);
+  const handelSelectAll = () => {
+    const selectedFiles = !selectAll ?  items.map((item) => { console.log(item); return item.fileId}): [];
+    setSelectAll(!selectAll);
+    console.log(selectedFiles);
+    handleFileSelect &&
+    handleFileSelect(selectedFiles);
+  };
+  const selectHandler = (fileId: string) => {
```
Comment (julienmachon) [inline apps/search-poc/src/components/ResultTable.tsx]: Can I ask why you haven't used the built in row selection of the Antd Table component? https://ant.design/components/table/#components-table-demo-row-selection

File: packages/nexus-sdk/src/Archive/index.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import { Fetchers, Resource } from '../types';
+import { NexusContext } from '../nexusSdk';
+import { ArchivePayload } from './types';
+
+const Archive = (
+  { httpGet, httpPut, httpPost, poll }: Fetchers,
+  context: NexusContext,
+) => {
+  return {
+    get: <T>(
```
Comment (julienmachon) [inline packages/nexus-sdk/src/Archive/index.ts]: According to the doc, there are a couple of ways you can retrieve an archive (content or metadata): https://bluebrainnexus.io/docs/api/current/kg/kg-archives-api.html#fetch-an-archive

I would add options, like we did for Resource https://github.com/BlueBrain/nexus-js/blob/master/packages/nexus-sdk/src/Resource/types.ts#L53 and https://github.com/BlueBrain/nexus-js/blob/master/packages/nexus-sdk/src/Resource/index.ts#L20-L26 with maybe `x-tar` being the default?

File: apps/search-poc/src/containers/ResourceDetails/RecNrnMorphologyCollection.tsx [hunk 5]
```diff
@@ -1,28 +1,33 @@
...
 import { parseProjectUrl, getLabel, camelCaseToLabelString } from '../../utils';
 import ResultTable from '../../components/ResultTable';
 import { HandleClickParams } from '../../types';
 
-
 const RecNrnMorphologyCollectionContainer: React.FunctionComponent<{
   resource: Resource & MINDSResource;
   handleClick: (params: HandleClickParams) => void;
+  isDownload: boolean;
+  OnFileSelect: (fileId: string[]) => void;
```
Comment (julienmachon) [inline apps/search-poc/src/containers/ResourceDetails/RecNrnMorphologyCollection.tsx]:
```suggestion
  onFileSelect: (fileId: string[]) => void;
```

File: packages/nexus-sdk/src/Archive/types.ts [hunk 6]
```diff
@@ -0,0 +1,11 @@
+export type ArchivePayload = {
+  resources: {
```
[the remark this answers is not in the document]
Comment (dhaneshnm) [reply packages/nexus-sdk/src/Archive/types.ts]: I have updated the code with preferred order.

**Pull request closed without merging**